tasks/db.yml

```diff
@@ -12,7 +13,48 @@
     query: |
       UPDATE okm_user
       SET usr_password = %s
-      WHERE usr_id = %s 
+      WHERE usr_id = %s
     positional_args:
       - '{{ openkm_admin_password | password_hash("bcrypt") }}'
       - '{{ openkm_admin_username | mandatory }}'
+
+- name: Set workflow.adapter config
+  postgresql_query:
+    db:             '{{ openkm_db_name | mandatory }}'
+    login_user:     '{{ openkm_db_user | mandatory }}'
+    login_password: '{{ openkm_db_pass | mandatory }}'
+    login_host:     'localhost'
+    query: |
+      UPDATE okm_config
+      SET cfg_value = 'com.openkm.workflow.adapter.JbpmRemoteWorkflowAdapter'
+      WHERE cfg_key = 'workflow.adapter'
+
+- name: Set workflow.adapter.password config
+  no_log: true
+  postgresql_query:
+    db:             '{{ openkm_db_name | mandatory }}'
+    login_user:     '{{ openkm_db_user | mandatory }}'
+    login_password: '{{ openkm_db_pass | mandatory }}'
+    login_host:     'localhost'
+    query: |
+      UPDATE okm_config
+      SET cfg_value = %s
+      WHERE cfg_key = 'workflow.adapter.pasword'
+    positional_args:
+      - '{{ openkm_jbpm_admin_password | mandatory }}'
+
+# TODO: jbpm doesn't like it, maybe due to bcrypt version 2a vs 2b
```

Comment:
What does "doesn't like it" mean? You can't log in or does it explode? Is there an error?

Reply:
You can't login with such password. I will rephrase it.